src/frontends/ir/docs/architecture.md

```diff
@@ -0,0 +1,45 @@
+# OpenVINO IR Frontend Architecture
+
+OpenVINO IR Frontend uses [pugixml](../../../thirdparty/pugixml/README.md) library to parse xml files. After that based on version and name of operation we found supported operation and create it using OpenVINO Visitor API:
```

Comment:
Could you please clarify who is we in the second sentence? Sounds a bit confusing